creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -1,4 +1,5 @@
-const versionInfo = require('../../../.version.json')
+const path = require('path')
+const versionInfo = require(path.join(process.cwd(), '.version.json'))
```

Comment:
these relative paths outside the src/ directory were causing issues because on the prod container it's path is build/src/ not src/ so it was throwing a file not found error

Reply:
nice find! Thank you